scripts/helper.js

```diff
@@ -145,17 +145,21 @@ const GetDepTree = ( name ) => {
 
 
 /**
- * Get all folders within a given path
+ * Get all modules within a given path.
```

Comment:
```suggestion
 * Get all modules within a given path
```

None of the other function descriptions have a full-stop, to be consistent we should leave it out